src/screens/Learn/Articles/Article.tsx

```diff
@@ -12,13 +12,37 @@ import {
   SmallSubtext,
 } from '../Learn.style';
 import { articlesById } from 'cms-content/articles';
+import SmallShareButtons from 'components/SmallShareButtons';
+import { trackEvent, EventAction, EventCategory } from 'components/Analytics';
 
 const Article = () => {
   let { articleId } = useParams<{ articleId: string }>();
   const metatagDate = formatMetatagDate();
   const articleContent = articlesById[articleId];
   const { header, body, date, summary } = articleContent;
 
+  const trackShareFacebook = () =>
+    trackEvent(EventCategory.ARTICLES, EventAction.SHARE, 'facebook');
+
+  const trackShareTwitter = () =>
+    trackEvent(EventCategory.ARTICLES, EventAction.SHARE, 'twitter');
+
+  const trackCopyLink = () => {
+    trackEvent(
+      EventCategory.ARTICLES,
+      EventAction.COPY_LINK,
+      'recommendations',
+    );
+  };
+
+  const shareButtonProps = {
+    shareUrl: `https://covidactnow.org/deep-dives/${articleId}`,
```

Comment:
Consider defining a `relativeUrl` variable and use it here and in the app meta tags to make sure that they are consistent.